source/scripting/Ped.hpp

```diff
@@ -587,8 +587,8 @@ namespace GTA
 		void RandomizeOutfit();
 		void SetDefaultClothes();
 		void LeaveGroup();
-		void Clone();
-		void Clone(float heading);
```

Comment:
It couldn't be changed til v3.0 branch though as it would break old scripts compiled on 2.X branches